Resources/Prototypes/_NF/Entities/Mobs/NPCs/pets.yml

```diff
@@ -0,0 +1,43 @@
+- type: entity
+  id: NFMobPet
+  abstract: true
+  components:
+  - type: GhostRole
+#  - type: IntrinsicRadioReceiver # DeltaV - comment out
+  - type: CargoSellBlacklist
+
+- type: entity
+  name: Clippy
+  parent: [NFMobPet, MobCatGhost]
+  id: MobCatClippy
+  description: It looks like you’re writing a letter, would you like help?
+  components:
+  - type: GhostRole
+    name: ghost-role-information-clippy-name
+    description: ghost-role-information-deltav-clippy-description # DeltaV - Alternate Description
+    rules: ghost-role-information-nonantagonist-rules
+    prototype: CatClippy
```

Comment:
Forgot to remove this ``prototype`` line.

Reply:
Woopsie